distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh

```diff
@@ -42,10 +41,11 @@ fi
 
 echo "--IGNORE SLF4J ERRORS"--
 
-RETURNCODE=$(java -cp "$JARFILE" -Djavax.net.ssl.keyStore="$KEYFILE" -Djavax.net.ssl.keyStorePassword="$PASSWORD" -Djavax.net.ssl.keyStoreType="$KEYTYPE" "$CLASSNAME" "$COMMONNAME")
```

Comment:
Updated the bash script. A copy and paste error from an old version wiped out the correct way to get the return code. I fixed that.